### DISCUSSION TITLE
Unable to run Sunbird Ed in local environment

### DISCUSSION THREAD

**Discussion by israr2800** [Sunbird-Ed/Community#381] [Installation] [answered]
Hi I'm unable to run this repository "https://github.com/Sunbird-Ed/SunbirdEd-portal" in local environment.

Steps I have followed:
1. Installed prerequisites in my system like: Docker and Kubernet
2. Followed all instructions mentioned on your portal "https://ed.sunbird.org/use-1/install-locally/sunbirded-portal"
3. What about the "environmentVariablesHelper.js"  configuration file? we haven't received till now, what can be done in this type of circumstances.

**Comment by Krishnaj20** [comment, marked as answer]
@vinukumar-vs - can you check and respond to this thread?

**Reply by Krishnaj20** [reply]
@rajeevsathish - can you look into this query and respond. If you are not able to answer please tag anyone else who can respond.

@vinukumar-vs

**Reply by rajeevsathish** [reply]
@israr2800 environmentVariablesHelper is the env variables that has to be added so that the local portal instance can be connected to the API and other services so all the value has to be updated depending on your instance specifications.

cc: @vinukumar-vs

**Reply by Krishnaj20** [reply]
@israr2800 - Pls let us know if your query is resolved. 

@gandham-santhosh @kameshbhr @anandvarada

**Reply by Krishnaj20** [reply]
@israr2800 - marking this thread as answered since we haven't heard back from you. Please feel free to raise a new thread for any follow up queries.

@gandham-santhosh @kameshbhr @anandvarada
